iothub/service/src/Registry2/ConfigurationsClient.cs

```diff
@@ -0,0 +1,460 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Common.Exceptions;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Subclient of <see cref="IotHubServiceClient"/> that handles creating, updating, getting and deleting configurations
```

Comment:
Oxford comma and end-of-sentence punctuation.

```suggestion
    /// Subclient of <see cref="IotHubServiceClient"/> that handles creating, updating, getting, and deleting configurations.
```